gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-healthcheck/pom.xml

```diff
@@ -109,6 +109,7 @@
             <groupId>io.netty</groupId>
             <artifactId>netty-tcnative-boringssl-static</artifactId>
             <classifier>${os.detected.classifier}</classifier>
```

Comment:
shouldn't we remove this line also ?

Reply:
no, it is for test purpose, si must be aligned with the host.